knx/knxnet/proto.go

```diff
@@ -133,6 +133,10 @@ func Unpack(data []byte, srv *Service) (uint, error) {
 	var headerLen, version uint8
 	var srvID ServiceID
 	var totalLen uint16
+	
+	if len(data) == 0 {
```

Comment:
This should be checked at the call site. Unpack is correct in returning io.UnexpectedEOF in situations where there is insufficient data.